src/libs/protocols/src/trans.rs

```diff
@@ -974,6 +977,18 @@ impl From<crate::oci::Hooks> for oci::Hooks {
         for hook in from.take_Prestart().to_vec() {
             prestart.push(hook.into())
         }
+        let mut create_runtime = Vec::new();
```

Comment:
While this is similar to the code that existed before, this is not super idiomatic. You could use something like:

```
        let create_container = from
            .take_CreateContainer()
            .into_iter()
            .map(|i| i.into())
            .collect();
```

This dispense with the need to make things `mut`.

Reply:
Great suggestion! Thanks!